project_config.yml

```
@@ -17,4 +17,11 @@ target: fare_amount
 id_col: pickup_zip
 
-mlflow_experiment_name: /Shared/mlops_course_annika
\ No newline at end of file
+mlflow_experiment_name: /Shared/mlops_course_annika
+
+ab_test:
+  learning_rate_a: 0.02
+  learning_rate_b: 0.02
+  n_estimators: 1000
+  max_depth_a: 6
+  max_depth_b: 10
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)